packages/client/mutations/EndDraggingReflectionMutation.ts

```diff
@@ -87,6 +87,42 @@ const handleAddReflectionGroupToGroups = (store, reflectionGroup) => {
   addNodeToArray(reflectionGroup, meeting, 'reflectionGroups', 'sortOrder')
 }
 
+const handleAddReflectionToSpotlight = (
+  reflection: RecordProxy,
+  reflectionGroup: RecordProxy,
+  oldReflectionGroupId: string,
+  store: RecordSourceSelectorProxy
+) => {
+  const reflectionGroupId = reflection.getValue('reflectionGroupId')
+  if (reflectionGroupId === oldReflectionGroupId) return
+  const meetingId = reflection.getValue('meetingId') as string
+  const meeting = store.get(meetingId)
+  if (!meeting) return
+  const spotlightReflection = meeting?.getLinkedRecord('spotlightReflection')
+  const spotlightReflectionId = spotlightReflection?.getValue('id')
+  const viewer = store.getRoot().getLinkedRecord('viewer')
+  if (!viewer || !spotlightReflectionId) return
+  const similarReflectionGroups = viewer.getLinkedRecords('similarReflectionGroups', {
+    reflectionId: spotlightReflectionId,
+    searchQuery: ''
+  })
+  const wasInSpotlightGroups = similarReflectionGroups?.find(
+    (group) => group.getValue('id') === oldReflectionGroupId
+  )
+  const isInSpotlightGroups = similarReflectionGroups?.find(
+    (group) => group.getValue('id') === reflectionGroupId
+  )
+  // check if ungrouping is happening in the spotlight
+  if (!isInSpotlightGroups && wasInSpotlightGroups) {
```

Comment:
If the following scenario happens:
- the Spotlight opens and has 10 reflection groups
- a user on another browser adds a reflection called "dogs" into one of the 10 Spotlight groups
- that user then removes the reflection "dogs" from that group

Would you expect the "dogs" reflection to remain in the Spotlight so that there are 11 groups or that it returns to the kanban? I think it should probably stay (which is how the current implementation works) because while it could have been a accidentally grouped, it's probably more likely that it's a relevant reflection group